src/acceptor_pool.erl

```diff
@@ -167,6 +168,7 @@ start_link(Name, Module, Args) ->
       Reason :: inet:posix().
 accept_socket(Pool, Sock, Acceptors)
   when is_port(Sock), is_integer(Acceptors), Acceptors > 0 ->
+    ok = gen_tcp:controlling_process(Sock, Pool),
```

Comment:
Technically its possible for socket to pass to the pool but the GenServer message not to because caller is killed by exit signal, this is the worst case as we get stuck with the pool holding the socket but knowing, can we move it to after the call?

Reply:
Yea, will do.